.gitignore

```
@@ -8,3 +8,5 @@ wheels/
 
 # Virtual environments
 .venv
+
+**/.DS_Store
```

```suggestion
.DS_Store
```

does this work just as well?